kernel/packages/decentraland-loader/lifecycle/controllers/scene.ts

```diff
@@ -44,11 +44,7 @@ export class SceneLifeCycleController extends EventEmitter {
     const parcels = this.sceneStatus.get(sceneId)?.sceneDescription?.sceneJsonData?.scene?.parcels
     if (parcels) {
       // Invalidate parcel to scenes association
-      for (const parcel of parcels) {
-        this._positionToSceneId.delete(parcel)
-        this.futureOfPositionToSceneId.delete(parcel)
-      }
-      this.downloadManager.invalidateParcels(parcels)
+      this.invalidateParcels(parcels)
```

Comment:
I really think that `this.sceneStatus.delete(sceneId)` should be part of the invalidation.